portfolio/src/main/webapp/script.js

```diff
@@ -25,3 +25,30 @@ function addRandomGreeting() {
   greetingContainer.innerText = greeting;
 }
 */
+
+
+/**
+ * Fetches greeting from the server and adds it to the DOM.
+ */
+function getGreeting() {
```

Comment:
Nit: It's more like a personal opinion but I think something like `fetchAndShowGreeting()` would be a more appropriate name for the method.

Reply:
That's a good point, the name doesn't seem to describe the method very accurately. Thanks!